packages/aws-cdk/lib/api/bootstrap/bootstrap-template.yaml

```diff
@@ -211,7 +211,10 @@ Resources:
           - Id: CleanupOldVersions
             Status: Enabled
             NoncurrentVersionExpiration:
-              NoncurrentDays: 30
+          - Id: AbortIncompleteMultipartUploads
+            Status: Enabled
+            AbortIncompleteMultipartUpload:
+              DaysAfterInitiation: 7
```

Comment:
Seems unlikely a CDK deployment is stuck for 7 days. The rule actually recommends 24h (i.e. 1 day). That seems much more reasonable to me.